cardano-api/src/Cardano/Api/TxBody.hs

```diff
@@ -290,17 +291,17 @@ instance IsCardanoEra era => ToJSON (TxOut era) where
       ShelleyAddressInEra sbe ->
         case sbe of
           ShelleyBasedEraShelley ->
-            let hexAddr = serialiseToRawBytesHexText addr
+            let hexAddr = serialiseToBech32 addr
```

Comment:
:+1:

Reply:
Since is now Bech32` we might want to rename the variable from `hexAddr`

Reply:
done